--- nginx.conf.old
+++ nginx.conf.new
@@ -40,7 +40,7 @@
 	# indicated by the request header Content-Length. If the stated content
 	# length is greater than this size, then the client receives the HTTP
 	# error code 413. Set to 0 to disable. Default is '1m'.
-	client_max_body_size 1m;
+	client_max_body_size 0;
 
 	# Sendfile copies data between one FD and other from within the kernel,
 	# which is more efficient than read() + write(). Default is off.
